src/pages/User/Settings/LearningSettings/useLearningSettings.ts

```diff
@@ -44,6 +44,29 @@ const useLearningSettings = () => {
     return `${hours}:${minutes}`;
   };
 
+  const fetchUserSettings = async () => {
+    try {
+      const response = await fetchWithToken({
+        endpoint: 'getUserSettings',
+        method: 'GET',
+      });
+      if (response.status === 200) {
+        const settings = response.response;
+        reset({
+          notifications: settings.notifications.map((notif: any) => ({
```

Comment:
wszystko z obiektu `settings` warto zdestrukturyzować

Reply:
useLearningSettings.ts:66 Error fetching user settings: 
TypeError: settings is not iterable
    at fetchUserSettings (useLearningSettings.ts:55:1)


Żeby to wyelimnować, trzeba dopisać jakieś sprawdzenie itp. 
Jest jakieś inne rozwiązanie?